packages/design-tokens/src/themes/heart.ts

```diff
@@ -188,11 +188,11 @@ export const heartTheme: Theme = {
       letterSpacing: "normal",
     },
     display0: {
-      fontFamily: '"Greycliff CF", "Noto Sans", Helvetica, Arial, sans-serif',
-      fontWeight: 700,
+      fontFamily: '"Tiempos Headline", Georgia, serif',
```

Comment:
I'm unsure myself, but should the fallback be Noto Serif Display, or should we add that in later?

> Noto Serif Display (Black) to become the default fallback for all instances of Tiempos Headline (Black)
https://docs.google.com/document/d/1G5iN3zZ0XHc0K5FKIFpa263nGW3l8d-jV2nWf6Gcucs/edit#

Reply:
That's a bit of a rabbit hole. I've added a card to triage: https://github.com/orgs/cultureamp/projects/5#card-55164189

IIRC last time we were concerned about the weight of the font (with all the glyphs we needed) and couldn't easily verify the changes.